src/simplewallet/simplewallet.cpp

```diff
@@ -3352,6 +3351,7 @@ bool simple_wallet::init(const boost::program_options::variables_map& vm)
         else
         {
           m_electrum_seed = "";
+          success_msg_writer() << "\nPlease enter the first 24 words of your electrum seed, and the seed date.\nIf using an old-style seed and don't remember the date you generated your wallet, write 2000-01-01.\n";
```

Comment:
Remove the bit about not remembering the date. It should be the user knows, and they supply the date otherwise I think the subsequent prompt already lets you know the date input is optional.